corporate/tests/test_remote_billing.py

```diff
@@ -1019,6 +1020,15 @@ def test_transfer_business_plan_from_server_to_realm(
             billing_schedule=CustomerPlan.BILLING_SCHEDULE_ANNUAL,
             tier=CustomerPlan.TIER_SELF_HOSTED_BUSINESS,
             status=CustomerPlan.ACTIVE,
+            automanage_licenses=True,
+        )
+        initial_license_count = 100
+        LicenseLedger.objects.create(
+            plan=server_plan,
+            is_renewal=True,
+            event_time=timezone_now(),
+            licenses=initial_license_count,
+            licenses_at_next_renewal=initial_license_count,
```

Comment:
Since we were already creating the `CustomerPlan` for this test, I did the same for the `LicenseLedger` object. I believe the original idea might have been to avoid adding/updating stripe test fixtures.